File: LATIHAN2DPBO2023/Java/Pilih.java
/*Saya Mochamad Khaairi NIM 2106416 mengerjakan Latihan 2
dalam mata kuliah Desain dan Pemrograman Berorientasi Objek 
untuk keberkahanNya maka saya tidak melakukan kecurangan 
seperti yang telah dispesifikasikan. Aamiin.*/

//import library
import java.util.Scanner;
import java.util.ArrayList;

//inisialisasi class Pilih
public class Pilih
{
	//atribut (hanya untuk tampungan inputan)
	String nik;
	String name;
	String gender;
	String univ;
	String email;
	String nim;
	String faculty;
	String major;
	int i, j;

	//membuat scanner
	Scanner sc = new Scanner(System.in);

	//constructor
	public Pilih()
	{
	}

	//method untuk menambahkan data
	public void add(ArrayList<Mahasiswa> list)
	{
		//instance objeck menggunakan class Mahasiswa
		Mahasiswa temp = new Mahasiswa();
		j = 0;
		//inputan untuk data mahasiswa
		System.out.print("Masukkan NIK:");
		nik = sc.next();
		System.out.print("Masukkan Nama:");
		name = sc.next();
		System.out.print("Masukkan jenis Kelamin:");
		gender = sc.next();
		System.out.print("Masukkan Asal Universitas:");
		univ = sc.next();
		System.out.print("Masukkan Email Universitas:");
		email = sc.next();
		System.out.print("Masukkan NIM:");
		nim = sc.next();
		System.out.print("Masukkan Fakultas:");
		faculty = sc.next();
		System.out.print("Masukkan Program Studi:");
		major = sc.next();

		//pengecekan apakah data sudah tersedia di list (berdasarkan nik)
		for(i = 0; i < list.size(); i++)
		{
			if(nik.equals(list.get(i).getNik()))
			{
				System.out.println("\nTidak dapat menambahkan data, NIK sudah tersedia!");
				j = 1;	
			}
		}

		//jika data tidak tersedia maka set inputan dan push ke list
		if(j == 0)
		{
			temp.setNik(nik);
			temp.setName(name);
			temp.setGender(gender);
			temp.setUniv(univ);
			temp.setEmail(email);
			temp.setNim(nim);
			temp.setFaculty(faculty);
			temp.setMajor(major);

			list.add(temp);
			System.out.println("\nData berhasil ditambahkan!");
		}
	}

	//method untuk menampilkan data
	public void show(ArrayList<Mahasiswa> list)
	{
		System.out.println("\nList:");
		for(i = 0; i < list.size(); i++)
		{
			System.out.println(Integer.toString(i + 1) + ".");
			System.out.println("NIK:" + list.get(i).getNik());
			System.out.println("Nama:" + list.get(i).getName());
			System.out.println("Jenis Kelamin:" + list.get(i).getGender());
			System.out.println("Asal Universitas:" + list.get(i).getUniv());
			System.out.println("Email Univeritas:" + list.get(i).getEmail());
			System.out.println("NIM:" + list.get(i).getNim());
			System.out.println("Fakultas:" + list.get(i).getFaculty());
			System.out.println("Prograsm Studi:" + list.get(i).getMajor());
		}
	}
}

File: LATIHAN2DPBO2023/PHP/Pilih.php
<!-- Saya Mochamad Khaairi NIM 2106416 mengerjakan Latihan 2
dalam mata kuliah Desain dan Pemrograman Berorientasi Objek 
untuk keberkahanNya maka saya tidak melakukan kecurangan 
seperti yang telah dispesifikasikan. Aamiin. -->

<?php
//import
require ('Human.php');

//inisialisasi class Pilih
class Pilih
{
	//constructor
	public function __construct()
	{
	}

	//method untuk menambahkan data
	public function add(&$list, $nik, $name, $gender, $univ, $email, $nim, $faculty, $major, $image)
	{
		//instance object menggunakan class Mahasiswa
		$mhs = new Mahasiswa();

		//set value dari parameter ke object
		$mhs->setNik($nik);
		$mhs->setName($name);
		$mhs->setGender($gender);
		$mhs->setUniv($univ);
		$mhs->setEmail($email);
		$mhs->setNim($nim);
		$mhs->setFaculty($faculty);
		$mhs->setMajor($major);
		$mhs->setImage($image);

		//push object ke list / array
		array_push($list, $mhs);
	}

	//method untuk menampilkan data
	public function show($list)
	{
		//output berupa tabel
		echo "<table border='1' cellspacing='0' cellpading='10' cellpadding='10'";
		echo "<tr>";
		echo "<th>No</th>";
		echo "<th>NIK</th>";
		echo "<th>Nama</th>";
		echo "<th>Jenis Kelamin</th>";
		echo "<th>Asal Universitas</th>";
		echo "<th>Email Universitas</th>";
		echo "<th>NIM</th>";
		echo "<th>Fakultas</th>";
		echo "<th>Program Studi</th>";
		echo "<th>Foto Profile</th>";
		echo "</tr>";
		$j = 1;
		$i;
		for($i = 0; $i < count($list); $i++)
		{
			echo "<tr>";
			echo  "<td>" . $j . '. ' . "</td>";
			echo "<td>" . $list[$i]->getNik() . "</td>";
			echo "<td>" . $list[$i]->getName() . "</td>";
			echo "<td>" . $list[$i]->getGender() . "</td>";
			echo "<td>" . $list[$i]->getUniv() . "</td>";
			echo "<td>" . $list[$i]->getEmail() . "</td>";
			echo "<td>" . $list[$i]->getNim() . "</td>";
			echo "<td>" . $list[$i]->getFaculty() . "</td>";
			echo "<td>" . $list[$i]->getMajor() . "</td>";
			echo "<td>" . "<img src='" . $list[$i]->getImage() . "' style='width: 100px;'>" . '</td>';
			echo "</tr>";
			$j = $j + 1;
		}
		echo "</table>";
		echo "<br>";
	}

	//destructor
	public function __destruct()
	{	
	}
}

?>